server/app/com/xsn/explorer/data/anorm/dao/BlockRewardPostgresDAO.scala

```diff
@@ -3,83 +3,62 @@ package com.xsn.explorer.data.anorm.dao
 import java.sql.Connection
 
 import anorm._
-import com.xsn.explorer.models.{
-  BlockReward,
-  BlockRewards,
-  PoSBlockRewards,
-  PoWBlockRewards,
-  RewardType,
-  TPoSBlockRewards
-}
+import com.xsn.explorer.models.{BlockReward, RewardType}
 import com.xsn.explorer.models.values.Blockhash
 import com.xsn.explorer.data.anorm.parsers.BlockRewardParsers.parseBlockReward
 
 class BlockRewardPostgresDAO {
+  type Reward = (BlockReward, RewardType, Option[BigDecimal], Option[Long])
```

Comment:
Use a case class instead, this is the same than the alias on `BlockRewardPostgresSerializer`.

Reply:
Also, `Option[BigDecimal], Option[Long]` are both present or none of them, another case class could help